src/components/FileUploader/FileUploader.tsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2017-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { useState } from 'react';
+import { css } from '@emotion/core';
+import { injectT, tType } from '@ndla/i18n';
+import { spacing } from '@ndla/core';
+import { UploadDropZone } from '@ndla/forms';
+import { uploadFile } from '../../modules/draft/draftApi';
+import { createFormData } from '../../util/formDataHelper';
+import handleError from '../../util/handleError';
+
+const wrapperCSS = css`
+  padding: 0 ${spacing.large};
+`;
+
+interface Props {
+  onFileSave: (files: { path: string; type: string; title: string }[]) => void;
+}
+
+const FileUploader = ({ onFileSave, t }: Props & tType) => {
+  const [saving, setSaving] = useState(false);
```

Comment:
Bør det eksplisitt si at state er av type boolean? `useState<boolean>(false)`

Reply:
Jeg mener ikke det. De to uttrykkene fungerer likt, den andre er bare litt mer eksplisitt.